scripts/tools/release-cli/release-cli.js

```diff
@@ -40,6 +40,7 @@ const pkgJsonPaths = {
   client: path.resolve(base, 'client', 'public', 'package.json'),
   authOAuth2: path.resolve(base, 'plugins', 'auth', 'public', 'auth-oauth2', 'package.json'),
   authOIDC: path.resolve(base, 'plugins', 'auth', 'public', 'auth-oidc', 'package.json'),
+  authOIDC_pkce: path.resolve(base, 'plugins', 'auth', 'public', 'auth-oidc-pkce', 'package.json'),
```

Comment:
It's not needed, because we have an own release cylce for this package, which means this script is not responsable for the release of auth-oidc-pkce.